<ui version="4.0" >
 <class>NccGui</class>
 <widget class="QWidget" name="NccGui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>NetCardConfig</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QTreeWidget" name="networkCardsTreeWidget" >
     <property name="tabKeyNavigation" >
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize" >
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="autoExpandDelay" >
      <number>-1</number>
     </property>
     <property name="indentation" >
      <number>30</number>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="headerHidden" >
      <bool>false</bool>
     </property>
     <property name="expandsOnDoubleClick" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Interface</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>IP address</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Description</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QPushButton" name="configPushButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Configure Interface</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="statusPushButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Enable Interface</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QPushButton" name="infoPushButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Interface Details</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="wlanPushButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Choose WLAN Network</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6" />
 <resources/>
 <connections>
  <connection>
   <sender>statusPushButton</sender>
   <signal>clicked()</signal>
   <receiver>NccGui</receiver>
   <slot>toggleCardStatus()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>292</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel" >
     <x>292</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>infoPushButton</sender>
   <signal>clicked()</signal>
   <receiver>NccGui</receiver>
   <slot>cardInfo()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>479</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel" >
     <x>292</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>networkCardsTreeWidget</sender>
   <signal>itemClicked(QTreeWidgetItem*,int)</signal>
   <receiver>NccGui</receiver>
   <slot>cardSelected()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>292</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel" >
     <x>292</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>configPushButton</sender>
   <signal>clicked()</signal>
   <receiver>NccGui</receiver>
   <slot>lanConfig()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>106</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel" >
     <x>292</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>networkCardsTreeWidget</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>NccGui</receiver>
   <slot>lanConfig()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>292</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel" >
     <x>292</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wlanPushButton</sender>
   <signal>clicked()</signal>
   <receiver>NccGui</receiver>
   <slot>wlanConfig()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>222</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel" >
     <x>292</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toggleCardStatus()</slot>
  <slot>cardInfo()</slot>
  <slot>cardSelected()</slot>
  <slot>lanConfig()</slot>
  <slot>wlanConfig()</slot>
 </slots>
</ui>
